02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>Some of the language here, including the variable name <code>unwrapped_coords</code> and the definition of <code>images</code> should be cleaned up as this example no longer unwraps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
good catch, I can remove freud from this example.